mender.go

```diff
@@ -315,7 +315,7 @@ func (m *mender) doBootstrap() menderError {
 }
 
 func (m *mender) FetchUpdate(url string) (io.ReadCloser, int64, error) {
-	return m.updater.FetchUpdate(m.api.Request(m.authToken), url)
+	return m.updater.FetchUpdate(m.api, url)
```

Comment:
@kacf download link does not need any tampering, if it will it's going to be documented. Links are signed by server thats why tweaking certain things invalidates it.